src/components/common/ActionSheet/index.js

```diff
@@ -0,0 +1,64 @@
+import i18next from 'i18next';
+import React from 'react';
+import { Text, View } from 'react-native';
+import { Modalize } from 'react-native-modalize';
+import { Portal } from 'react-native-portalize';
+
+import Touchable from '../Touchable';
+import styles from './styles';
+
+function ActionSheet({ modalRef, onClose, title, subtitle, options }) {
+  const handleClose = () => {
+    modalRef?.current?.close();
+    onClose?.();
+  };
+
+  const handleItemPress = option => {
+    option?.onPress();
+    handleClose();
+  };
+
+  return (
+    <Portal>
+      <Modalize
+        ref={modalRef}
+        withHandle={false}
+        adjustToContentHeight
+        modalStyle={styles.modal}>
+        <View>
+          <View style={styles.topContainer}>
+            {(title || subtitle) && (
+              <View style={styles.titleContainer}>
+                {title && <Text style={styles.title}>{title}</Text>}
+                {subtitle && <Text style={styles.subtitle}>{subtitle}</Text>}
+              </View>
+            )}
+            {options?.map((option, index) => (
```

Comment:
```suggestion
            {options?.map(({ id, label, destructive }, index) => (
```

Reply:
im using the whole `options` object below 👼  in handlePress